src/Utility/Constants.ts

```diff
@@ -196,6 +196,41 @@ export const allowedEmails = [
   'graduate.org',
   'musician.org',
   'teachers.org',
+  'shiba.bar',
+  'protonmail.com',
+  'aol.com',
+  'mailcow.email',
+  'cock.li',
+  'discord.com',
+  'googlemail.com',
+  'yahoo.com',
+  'zoho.com',
+  'zohomail.com',
+  'privatemail.com',
+  'ionos.com',
+  'emailonacid.com',
+  'squirrelmail.org',
+  'airmail.cc',
+  'modoboa.org',
+  '420blaze.it',
+  'aaathats3as.com',
+  'cumallover.me',
+  'goat.si',
+  'national.shitposting.agency',
+  'loudwashere.org',
+  'tfwno.gf',
+  'cock.lu',
+  'cock.email',
+  'firemail.cc',
+  'memeware.net',
+  'cocaine.ninja',
+  'waifu.club',
+  'dicksinhisan.us',
+  'loves.dicksinhisan.us',
+  'wants.dicksinhisan.us',
+  'dicksinmyan.us',
+  'loves.dicksinmyan.us',
```

Comment:
```suggestion
  'goat.si',
  'national.shitposting.agency',
  'tfwno.gf',
  'firemail.cc',
  'memeware.net',
  'waifu.club',
```

drop the inappropriate emails